ush/ioda/bufr2ioda/run_bufr2ioda.py

```diff
@@ -30,8 +31,9 @@ def bufr2ioda(current_cycle, RUN, DMPDIR, config_template_dir, COM_OBS):
     }
 
     # Specify observation types to be processed by a script
-    BUFR_py = ["satwind_amv_goes", "satwind_scat", "adpupa_prepbufr", "adpsfc_prepbufr", "sfcshp_prepbufr", "acft_profiles_prepbufr",
-               "gpsro_bufr", "conventional_prepbufr_ps"]
+    BUFR_py_file = glob.glob(os.path.join(USH_IODA, 'bufr2ioda_*.py'))
+    BUFR_py_files = [os.path.basename(f) for f in BUFR_py_files]
```

Comment:
Should the second occurrence of `BUFR_py_files` be `BUFR_py_file`?  That is, should this line read

`BUFR_py_files = [os.path.basename(f) for f in BUFR_py_file]`

Reply:
good catch, I would advocate for the opposite, make the first line `BUFR_py_files`. I'll fix.